.github/workflows/release.yml

```
@@ -0,0 +1,104 @@
+        echo "1. Download the \`bvim-${{ steps.get_version.outputs.version }}.zip\` file" >> release_notes.md
+        echo "2. Extract the zip file" >> release_notes.md
+        echo "3. Open Chrome and go to \`chrome://extensions/\`" >> release_notes.md
+        echo "4. Enable Developer mode" >> release_notes.md
+        echo "5. Click 'Load unpacked' and select the extracted folder" >> release_notes.md
+        
+        echo "Generated release notes:"
+        cat release_notes.md
+    
+    - name: Create GitHub Release
+      if: steps.check_tag.outputs.exists == 'false'
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)